dashboard/dashboard-front/src/components/jobs/JobDetails.vue

```diff
@@ -26,9 +26,11 @@ const logsOpen: Ref<boolean> = ref(false)
 const loadingLogs: Ref<boolean> = ref(false)
 const manifestDialogRef: Ref<typeof ManifestEditDialog | null> = ref(null)
 
-const manifestYaml: Ref<string> = computed(() => 
-    yaml.dump(removeNulls(job.value?.manifest)) || ''
-)
+const manifestYaml: Ref<string> = computed(() => {
+    if (!job.value?.manifest)
```

Comment:
This is if a job does not have a manifest, correct? This doesn't seem like something that'd happen under normal use, right? If so, it's maybe worth logging a warning.